packages/analytics/analytics-chart/src/components/chart-plugins/ChartTooltip.vue

```diff
@@ -105,15 +182,31 @@ watch(tooltipEl, value => {
   box-shadow: 0 1px 3px rgba(0, 0, 0, 0.12), 0 1px 2px rgba(0, 0, 0, 0.24);
   max-width: 425px;
   min-width: 250px;
-  pointer-events: none;
+  overflow-y: scroll;
   position: absolute;
+  scrollbar-width: thin;
   transition: all 0.3s cubic-bezier(0.25, 0.8, 0.25, 1);
   z-index: 999;
+
+  &::-webkit-scrollbar-track {
+    background-color: var(--white);
```

Comment:
this might seem like overkill, but can we pls add a fallback:

`background-color: var(--white, #ffffff);`